plugins/org.eclipse.emf.mwe.core/src/org/eclipse/emf/mwe/internal/core/debug/processing/handlers/EventRuntimeHandler.java

```diff
@@ -83,7 +83,7 @@ public void started() throws IOException {
 	/**
 	 * Push the element onto the stack
 	 * 
-	 * @see org.eclipse.emf.mwe.internal.core.debug.processing.EventHandler#preTask(java.lang.Object,
+	 * @see org.eclipse.emf.mwe.core.debug.processing.EventHandler#preTask(java.lang.Object,java.lang.Object,
```

Comment:
why qualifying Object here?

Reply:
i did not change the existing pattern. i just fixed places inline to be consistent
=> please open separate pr

Reply:
not worth the effort